modules/loader.js

```diff
@@ -459,8 +459,8 @@ __private.loadBlockChain = function() {
 	 */
 	function checkMemTables(t) {
 		const promises = [
-			t.blocks.count(),
-			t.blocks.getGenesisBlock(),
+			library.storage.entities.Block.count({}, t),
```

Comment:
I guess we decided to follow the convention of three parameters in all CRUD operations. So please add the `options` as second parameter in operation and then use `{}` here.